docs/introduction.md

```diff
@@ -0,0 +1,29 @@
+Studio is a tool to develop AsyncAPI documentation. 
+
+Using Studio you can develop an AsyncAPI document, visualize the flow of events, preview, validate, and convert the document to the latest version in one place.
```

Comment:
I'll change `develop` here by `create`, as we don't really develop a document.
`convert the document` -> `convert the AsyncAPI document`

Reply:
Not sure why Jagruti didn't make this suggested change, but I will do ahead and add it. :)